src/django/api/templates/mail/report_result_body.txt

```diff
@@ -1,16 +1,30 @@
 {% block content %}
-Hi,
-
-We’re writing to notify you that your report that {{ facility_name }} has been
-{{ closure_state|lower }} has been {{ status|lower }}.
+    Hi there,
 
+    Thanks for reporting {{ facility_name }} facility as {{ closure_state|lower }}.
+{% if is_closure and is_rejected %}
+    We have rejected this report and the facility's profile in the OAR has not been changed.
+{% endif %}
```

Comment:
Newlines in the templates are preserved in the output, even if the only content on the line is a template tag

```
django_1    | --===============8035258767979269043==
django_1    | Content-Type: text/plain; charset="utf-8"
django_1    | MIME-Version: 1.0
django_1    | Content-Transfer-Encoding: 7bit
django_1    |
django_1    |
django_1    |     Hi there,
django_1    |
django_1    |     Thanks for reporting 4teams - Advertising &amp; Merchandising, Lda. facility as closed.
django_1    |
django_1    |     We have rejected this report and the facility's profile in the OAR has not been changed.
django_1    |
django_1    |
django_1    |
django_1    |
django_1    |
django_1    |     OAR Team review notes: NOO
django_1    |
django_1    |
django_1    |     If you have additional evidence that can be used to verify the facility has closed, this can be shared with: info@openapparel.org
django_1    |
django_1    |
django_1    |
django_1    |     Best wishes,
django_1    |
django_1    |
django_1    | Open Apparel Registry
django_1    |
django_1    |
django_1    |
django_1    |
django_1    | --===============8035258767979269043==
```
It makes the conditionals in the templates a little harder to read but we should consider removing some newlines

Reply:
Updated: 
<img width="844" alt="Screen Shot 2021-02-25 at 10 23 27 AM" src="https://user-images.githubusercontent.com/21046714/109177764-26fdeb80-7756-11eb-9bcb-fe4a020588d7.png">